phpunit.xml.dist

```diff
@@ -6,12 +6,26 @@
       <directory>src</directory>
     </include>
     <exclude>
+      <!-- Internal -->
       <file>src/preload.php</file>
       <file>src/bootstrap.php</file>
       <directory>src/Psl/Internal</directory>
+
+      <!-- Base Exceptions -->
       <directory>src/Psl/Exception</directory>
+
+      <!-- Constants -->
       <file>src/Psl/Str/constants.php</file>
       <file>src/Psl/Math/constants.php</file>
+      <file>src/Psl/Filesystem/constants.php</file>
+
+      <!-- Things that are not easily tested -->
+      <file>src/Psl/Filesystem/get_group.php</file>
+      <file>src/Psl/Filesystem/change_group.php</file>
+      <file>src/Psl/Filesystem/Internal/change_group.php</file>
+      <file>src/Psl/Filesystem/get_owner.php</file>
+      <file>src/Psl/Filesystem/change_owner.php</file>
+      <file>src/Psl/Filesystem/Internal/change_owner.php</file>
```

Comment:
no way for us to really test these.

Reply:
Other packages do this by not using fully-qualified function calls.
This way you can overwrite the internal php functions in the namespace of the function and assert based on the attributes passed to that function.

A bit dirty and it doesn't  really test the result, but it gives you more than not testing...

Reply:
i would rather not test it, otherwise we are doing a hack just to make it seem like we are covering it, but we are not actually.